### PR TITLE
Localize “email address” placeholder on home page

### PR DESCRIPTION
**Pull request by npu21**
https://github.com/kubernetes/website/issues/19361

### CODE REVIEW

File: i18n/ja.toml [hunk 1]
```diff
@@ -186,3 +186,6 @@ other = "次の項目"
 
 [warning]
 other = "警告:"
+
+[input_placeholder_email_address]
+other = "電子メールアドレス"
```
Comment (inductor) [inline i18n/ja.toml]:
```suggestion
other = "メールアドレス"
```

**Comment by sftim** [comment]
Someone with access can /approve this if someone from @kubernetes/sig-docs-ja-owners adds /approve to indicate it's OK

**Pull request merged**